### DISCUSSION TITLE
1.3RC1

### DISCUSSION THREAD

**Discussion by Marc2480** [BimmerGestalt/AAIdrive#308] [General]
Congratulations Walter with the release candidate!

Last couple of weeks the (almost) daily builds have seen big improvements (at least for me ;)) in stability and usability!

Keep up the good work and thanks for this amazing tool!!

Marc2480

**Comment by fookyuanloh** [comment]
Are we putting this on on Google PlayStore yet?
Facing difficulty to install apps that are not via/thru PlayStore as my android device is under the MDM management (corporate emails etc), and it disallow any direct apk installation

**Reply by hufman** [reply]
Not yet, that's scheduled as part of #212 👍

**Comment by Marc2480** [comment]
want to add this, have been running the 1.3RC2 June 14 build now for ~2 days, and to me it looks as if the "tunnel" (if I have understood correctly) to  Spotify is now correctly closed all the times when leaving the car. I am under impression that this is now also fixing the loosing access to the Spotify library (or searching) that happened earlier quiet often ...
Will continue using 1.3RC2 for a few more days to confirm this early conclusion ...
Again, amazing job!!!!

**Reply by donsaimon** [reply]
I experienced yesterday the loosing access to the Spotify library (or searching)...

**Reply by Marc2480** [reply]
So now on 1.3.RC2-8 for a few days and must say, rock-solid !!!!!

**Reply by hufman** [reply]
Thanks for the feedback!
Spotify has several intermittent problems, which are frustrating both as a user and as a developer. It sounds like you experienced #254, which is still open. I believe #254 and #217 will be improved by fixing #173, which is scheduled to be fixed as part of this release.
